k8s/cassandra/apptest/deployer/manifest/tester.yaml.template

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```

Comment:
You were absolutely right in our discussion--we should have a flavor of testrunner in which a single test-suite.yaml is the only thing that you need to add the test to the deployer. Please do file a FR issue in tools

Reply:
Added https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/issues/302